cmd/assetsvc/handler.go

```diff
@@ -273,33 +273,6 @@ func listChartsWithFilters(w http.ResponseWriter, req *http.Request, params Para
 	response.NewDataResponse(cl).Write(w)
 }
 
-// searchCharts returns the list of charts that matches the query param in any of these fields:
-//  - name
-//  - description
-//  - repository name
-//  - any keyword
-//  - any source
-//  - any maintainer name
-func searchCharts(w http.ResponseWriter, req *http.Request, params Params) {
```

Comment:
this will be needed if we implement the pagination (#1010) but we can recover the code in that case.